components/snackbar.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-snackbar v-model="open" :timeout="3000" :color="variant">
+    <span :style="{ color: textColor }">{{ message }}</span>
+    <v-btn :style="{ color: textColor }" text @click="open = false"
+      >Close</v-btn
+    >
+  </v-snackbar>
+</template>
+
+<script lang="ts">
+export default {
+  data() {
+    return {
+      open: false,
+      variant: '',
+      message: '',
+    }
+  },
+  computed: {
+    textColor: function () {
+      const variantToTextColorMap = {
```

Comment:
I've usually put static maps inside the `data()` function (or inside `~/services/constants.js` and them imported them), do you think one of these ways might be better, performance-wise?

Reply:
Yeah technically having it in the data is better in this case, since now it's going to recreate the map object every time the function is called. Realistically though, this is never really be an issue.
But we could move it, for correctness' sake!

I'd only move it to another file if we start using it somewhere else too.

Reply:
Yeah the performance implications are probably close to nothing, but may as well have it in the data() function :+1:

Reply:
Yup, will fix that in a PR tomorrow! 😄